frontend/modules/viewer/viewer.sagas.ts

```diff
@@ -30,7 +31,7 @@ export function* waitForViewer() {
 		const ViewerService = yield getAngularService('ViewerService') as any;
 		yield ViewerService.initialised.promise;
 	} catch (error) {
-		console.error(error);
+		yield put(DialogActions.showErrorDialog('initialise', 'viewer', error));
```

Comment:
- If the error is an enpoint error should use the DialogActions.showEndpointErrorDialog action instead. If it is a string its ok. Please check every call.